.github/workflows/binaries.yml

```diff
@@ -0,0 +1,55 @@
+name: Build binaries
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        required: true
+        type: string
+  workflow_call:
+    inputs:
+      version:
+        required: true
+        type: string
+
+jobs:
+  binaries:
+    name: Build lodestar binaries
+    strategy:
+      matrix:
+        include:
+          - os: ubuntu-latest
+            build: |
+              npx caxa -D -p "yarn install --production" --input . --output "lodestar" -- "{{caxa}}/node_modules/.bin/node" "--max-old-space-size=4096" "{{caxa}}/node_modules/.bin/lodestar"
+              tar -czf "dist/lodestar-linux-${{ inputs.version }}.tgz" "lodestar"
+    runs-on: ${{matrix.os}}
+    steps:
+      - uses: actions/checkout@v4
+      - uses: "./.github/actions/setup-and-build"
+        with:
+          node: 20
+      - run: |
+          mkdir -p dist
+          yarn global add caxa@3.0.1
+          ${{ matrix.build }}
+      - name: Upload binaries
+        uses: actions/upload-artifact@v4
+        with:
+          name: binaries-${{ matrix.os }}
+          path: dist/
+          if-no-files-found: error
+      - name: Sanity check binary
+        uses: actions/github-script@v7
+        with:
+          script: |
+            exec.exec('./lodestar beacon');
```

Comment:
this will by default start a mainnet node, might be better to start a `dev` node, it would also start the validator client and beacon node